tests-user/tests/create-lz-oapp.bats

```diff
@@ -211,6 +222,17 @@ teardown() {
     yarn lint:fix
 }
 
+@test "should work with yarn & native-oft-adapter example in CI mode" {
+    local DESTINATION="$PROJECTS_DIRECTORY/yarn-oapp"
```

Comment:
Should this be `yarn-native-oft-adapter` instead? I copied this over from the `oft-adapter` test right above which also lists the destination as `yarn-oapp`. I can fix both if this is a typo